welcome/run-d-program-locally.md

```diff
@@ -1,37 +1,39 @@
-# D Programme lokal ausführen
+# D-Programme lokal ausführen
 
-Eine D Installation beinhaltet unter anderem den D Compiler `dmd`,
+Eine Installation von D beinhaltet unter anderem den D-Compiler `dmd`,
 ein Skripttool `rdmd` und den Paketmanager `dub`.
 
-### DMD Compiler
-
-Der *DMD* Compiler kompiliert D Dateien und erstellt eine Binärdatei.
-Auf der Kommandozeile kann *DMD* mit dem Dateinamen aufgerufen werden:
+### Compiler DMD
+DMD (Digital Mars D) ist der Referenz-Compiler von D und wird für den Einstieg
```

Comment:
's gibt es im Deutschen fast nicht: https://de.m.wikipedia.org/wiki/Apostroph

Eigennamen sind die einzigen Ausnahmen, und sowohl "Ds" Referenz-Compiler als auch "D's" sind richtig und wahrscheinlich ist in diesem Fall sogar der Apostroph hilfreich, da er "D" als unbekannten Eigennamen betont.